src/services/codefixes/importFixes.ts

```diff
@@ -945,11 +954,9 @@ namespace ts.codefix {
                     importKind === ImportKind.Named ? [{ name: symbolName, addAsTypeOnly }] : emptyArray,
                     compilerOptions);
                 const moduleSpecifierWithoutQuotes = stripQuotes(moduleSpecifier);
-                return [
-                    importKind === ImportKind.Default ? Diagnostics.Add_default_import_0_to_existing_import_declaration_from_1 : Diagnostics.Add_0_to_existing_import_declaration_from_1,
-                    symbolName,
-                    moduleSpecifierWithoutQuotes
-                ]; // you too!

```

Comment:
you too!

Reply:
I left it in for so long, never knowing what it was referencing.